docker/local-single/docker-compose.yml

```diff
@@ -1,5 +1,11 @@
 version: "3.3"
 services:
+    zaas-service:
+        image: ghcr.io/zowe/zaas-service:latest
+        volumes:
+            - ../../config/docker/api-defs-shared:/api-defs
```

Comment:
Why zaas is needed in the volume containing api definitions?

Reply:
It is not needed - removed